client/src/assets/stylesheets/components/File.scss

```diff
@@ -1,79 +1,75 @@
 .file {
-    width: 20%;
-    display: grid;
-    padding-top: 30px;
-    padding-bottom: 15px;
-    margin: 0 auto;
-    background-image: url('../../images/lab7/FileLarge.png');
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: contain;
+  min-width: 220px;
+  padding-top: 30px;
+  padding-bottom: 15px;
+  margin: 0 auto;
+  background-image: url("../../images/lab7/FileLarge.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain;
 }
 
 .accessStatus {
-    width: 50px;
-    padding: 20px;
+  width: 50px;
+  padding: 20px;
 }
 
 .fileInfo {
-    text-align: center;
-    height: fit-content;
-    margin: auto;
+  text-align: center;
+  height: fit-content;
+  margin: auto;
 }
 
 @media screen and (min-width: 786px) {
-    .fileName {
-        font-size: 14px;
-    }
+  .fileName {
+    font-size: 14px;
+  }
 
-    .sensitivityLevel {
-        font-size: 16px;
-    }
+  .sensitivityLevel {
+    font-size: 16px;
+  }
 
-    .content {
-        font-size: 80%;
-    }
+  .content {
+    font-size: 80%;
+  }
 
-    .accessStatus {
-        padding: 10px;
-
-    }
+  .accessStatus {
+    padding: 10px;
```

Comment:
Just a little nit-picky thing but it might be worth using rem or em instead of px but not a dire change

Reply:
Will keep it in mind for future reference. Files have been removed because tailwind was used.